zowe_conformance/support_provider_evaluation_guide_table.md

```diff
@@ -1,115 +1,107 @@
-# Zowe Support Provider Conformance Guide
+# Zowe Support Provider v2 Conformance Guide
```

Comment:
Same here - the program name is the same, just the version is different

Reply:
Resolved